src/semgrep_agent/meta.py

```diff
@@ -223,21 +252,16 @@ def event_name(self) -> str:
         return os.getenv("CI_PIPELINE_SOURCE", "unknown")
 
     @cachedproperty
-    def mr_id(self) -> Optional[str]:
+    def pr_id(self) -> Optional[str]:
         return os.getenv("CI_MERGE_REQUEST_IID")
 
     @cachedproperty
-    def mr_title(self) -> Optional[str]:
+    def pr_title(self) -> Optional[str]:
         return os.getenv("CI_MERGE_REQUEST_TITLE")
 
     def to_dict(self) -> Dict[str, Any]:
```

Comment:
Should we remove the override since this is behavior of base class?

Reply:
yes, good catch

Reply:
wait, are you talking about the to_dict method?

Reply:
Yeah? It looks like the to_dict is just a pass through to the parent class?

Reply:
Ok, thanks for the clarification. Just fixed... GH interface shows the 4 lines above where you commented and it highlights the diffs instead of the line you commented on, so looked misleading.